src/Microsoft.Azure.WebJobs.Host/Tables/TableExtension.cs

```diff
@@ -306,43 +305,6 @@ public JObjectBuilder(TableExtension bindingProvider)
             }
         }
 
-        // IStorageTable --> IQueryable<T>
-        // ConverterManager's pattern matcher will figure out TElement. 
-        private class TableToIQueryableConverter<TElement> :
```

Comment:
The storage SDK does not support `IQueryable` on the version used here. If we wanted to support it, we'd have to provide our own implementation, which I would advise against (that's also the feedback I got from them and other teams). I believe this is a _point in time_ issue and `IQueryable` support will eventually land there.